packages/lesswrong/lib/collections/comments/views.ts

```diff
@@ -559,6 +561,12 @@ Comments.addView('shortformFrontpage', (terms: CommentsViewTerms) => {
             relevantTagIds: terms.relevantTagId,
           }
           : {},
+        {
+          $or: [
+            {reviewedByUserId: {$exists: true}},
+            {userId: currentUserId},
```

Comment:
What is this meant to be doing?  Right now it seems like it's checking either that a shortform comment was reviewed, or that the user who's looking at it is the one who posted it?

Reply:
Ah, wrong field. I bothered to check and fixed now.